### PR TITLE
WT-10881 Add simple fixed len column store testing for predictable replay in test/format

### PR DESCRIPTION
**Pull request by ddanderson**

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -525,9 +530,9 @@ functions:
         config=../../../test/format/CONFIG.replay
         for i in $(seq ${times}); do
           echo Iteration $i/${times}
-          x2=$RANDOM$RANDOM
-          x3=$RANDOM$RANDOM
-          rm -rf RUNDIR_{1,2,3}
```
Comment (ddanderson) [inline test/evergreen.yml:530]: Evergreen `shell.exec` uses /bin/sh.  That means that `$RANDOM` is not available, and also the `{a,b,c}` syntax is not expanded.

File: test/evergreen.yml [hunk 2]
```diff
@@ -510,6 +510,11 @@ functions:
         # replay using the same data seed, should guarantee we have equivalent data created.
         set -o errexit
         set -o verbose
+        # Get a random value with leading zeroes removed, /bin/sh version.
+        rando() {
+          tr -cd 0-9 </dev/urandom | head -c 5 | sed -e 's/0*\(.\)/\1/'
+        }
```
Comment (keitharnoldsmith) [inline test/evergreen.yml:516]: Nice!

**Review by keitharnoldsmith — APPROVED** [review]
LGTM!

**Comment by ddanderson** [comment]
evergreen merge

**Pull request merged**